### DISCUSSION TITLE
[Projects Beta] Server Error When Closing Personal Projects

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#16594] [Projects and Issues] [answered]
Whenever I close a personal project, it redirects to the project page, which gives me a 500 error code page. When I refresh the page, it gives me a 404. This is probably not desirable behavior.

**Comment by mariorod** [comment, marked as answer]
Thanks for reporting this ... we are investigating and will provide an update shortly

**Reply by shiftkey** [reply]
A fix for this was deployed yesterday which should resolve this for personal projects! Thanks for the report @PythonCoderAS!
